specification/deviceregistry/resource-manager/Microsoft.DeviceRegistry/preview/2023-11-01-preview/asset.json

```diff
@@ -0,0 +1,663 @@
+{
```

Comment:
FYI, on the Bluefin side we took a dependency on CADL/Typespec - I believe ARM is pushing for this. It makes maintaining & authoring OpenAPI specs easier. 

https://github.com/Azure/typespec-azure/tree/main/packages/typespec-providerhub-templates

Reply:
Yes migration to CADL is on the roadmap and after ignite release.